robusta_krr/core/integrations/prometheus/metrics_service/victoria_metrics_service.py

```diff
@@ -42,6 +42,10 @@ class VictoriaMetricsService(PrometheusMetricsService):
 
     service_discovery = VictoriaMetricsDiscovery
 
+    @classmethod
+    def name(cls) -> str:
+        return "Victoria Metrics"
```

Comment:
do we need this function also in PrometheusMetricsService and ThanosMetricsService?

Reply:
No, by default it parses it from the class name
With `VictoriaMetricsLoader` it parsed it as `Victoria`, so had to change it as it was confusing